TombEngine/Specific/Input/InputAction.h

```diff
@@ -68,7 +63,7 @@ namespace TEN::Input
 	{
 	private:
 		// Members
-		ActionID ID				= In::None;
+		ActionID ID				= In::Forward;
```

Comment:
Why this change was made?

Reply:
`None` is an invalid action that was used in a GUI input check, but it's no longer necessary.